crates/diesel_models/src/query/events.rs

```diff
@@ -222,4 +206,94 @@ impl Event {
         )
         .await
     }
+
+    fn apply_filters<T>(
+        mut query: T,
+        profile_id: Option<common_utils::id_type::ProfileId>,
+        from_to: (
+            dsl::created_at,
+            time::PrimitiveDateTime,
+            time::PrimitiveDateTime,
+        ),
```

Comment:
```suggestion
        (column, created_after, created_before): (
            dsl::created_at,
            time::PrimitiveDateTime,
            time::PrimitiveDateTime,
        ),
```

This would help avoid the tuple field access using `.0`, `.1` and `.2`.